pkg/processor/trigger/rabbitmq/test/rabbitmq_test.go

```diff
@@ -242,6 +242,8 @@ func (suite *testSuite) invokeEventRecorder(createFunctionOptions *platform.Crea
 		suite.Require().NoError(err, "Failed to unmarshal response")
 
 		// compare bodies
+		suite.T().Logf("sent    : %+v\n", sentEventBodies)
```

Comment:
There's a nuclio logger in there - please use that in the tests

Reply:
Done